src/renderer/components/dashboardPage/issues/index.vue

```diff
@@ -46,11 +46,12 @@
         @click="logout"
       />
     </div>
-    <perfect-scrollbar ref="issuesContainer" class="issues">
+    <div ref="issuesContainer" class="issues">
```

Comment:
Rozumiem że teraz będzie brzydki scroll na Windowsie i Linuxie?
w dokumentacji piszą że spokojnie można skorzystać z natywnego scrollTo i powinno zadziałać. Ale chyba to nie zadziałało, jeśli wywaliłeś xD